common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -440,34 +440,6 @@ case class InvokerResourceMessage(status: String,
    * Serializes message to string. Must be idempotent.
    */
   override def serialize: String = InvokerResourceMessage.serdes.write(this).compactPrint
-
-  def canEqual(a: Any) = a.isInstanceOf[InvokerResourceMessage]
-
-  override def equals(that: Any): Boolean =
-    that match {
-      case that: InvokerResourceMessage => {
-        that.canEqual(this) &&
-        this.status == that.status &&
-        this.freeMemory == that.freeMemory &&
-        this.busyMemory == that.busyMemory &&
-        this.inProgressMemory == that.inProgressMemory &&
-        this.tags.toSet == that.tags.toSet
-        this.dedicatedNamespaces.toSet == that.dedicatedNamespaces.toSet
-      }
-      case _ => false
-    }
-
-  override def hashCode: Int = {
```

Comment:
@ningyougang 
IIRC, we introduced this not to store data to ETCD if the data is the same as the previous one.
Can we make sure this requirement with the default comparison functionality provided by the case class?

I am not quite sure but we were using this class without these methods but introduced them at some point for some reason.

Could you confirm once?

Reply:
Ok, i will check